Samples/Swift/DaysUntilBirthday/Shared/Services/VerificationLoader.swift

```diff
@@ -124,14 +124,11 @@ final class VerificationLoader: ObservableObject {
                                   completion: @escaping () -> Void) {
     self.verificationPublisher(verifyResult: verifyResult) { publisher in
       self.cancellable = publisher.sink { sinkCompletion in
-        switch sinkCompletion {
-        case .finished:
-          break
-        case .failure(let error):
+        if case .failure(let error) = sinkCompletion {
           self.verification = Verification.noVerificationSignal
           print("Error retrieving age verification: \(error)")
-          completion()
         }
+        completion()
```

Comment:
I see two completions here. Does that mean we will call `completion()` _both_ when a value is received and when the `sink` completes? If so, should we call `completion()` twice?

Reply:
In the case of a failure or succession case, completion should only be called once. Looks like in this case it's being called twice. I moved the the completion to be within the `.failure` check but I believe this method also works if I only have the `completion()` at line 131 and remove the one at line 134 (now line 136).